.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+### Expected behaviour
```

Comment:
We should definitely force people to include the full laravel version and php version here.